jupyterbook/content/code_gallery/data_access_notebooks/2017-01-18-siphon-explore-thredds.ipynb

```diff
@@ -11,7 +11,9 @@
     "[Siphon](http://siphon.readthedocs.io/en/latest/) is a Python module for accessing data hosted on a THREDDS data server.\n",
```

Comment:
<p>I get this error message when running in Jupyter:</p><p><code>WebDriverException: Message: 'geckodriver' executable needs to be in PATH.&nbsp;</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ioos/ioos_code_lab/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Oh. That is odd :-/ 

I updated the page to the final redirected one anyway. Hopefully that helps.

Reply:
I guess this comment was added to the wrong place in the code. 

The issue I run into is the last cell `Image()` fails because I'm missing some driver `geckodriver`. See attached screenshot:

![image](https://user-images.githubusercontent.com/8480023/154109169-9d7eca0e-4368-43a1-b425-771e979b45dc.png)

Reply:
<p>That is a folium installation problem. No need to run that cell though, it is there only to capture the map as an image for the thumbnail.</p>